pkg/hubble/observer/local_observer.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/mennanov/fmutils"
```

Comment:
How much code are we using from this package, and how difficult would it be to avoid importing this?

Reply:
We are using most of the code in the package (which is extremely small). As we don't have any special use-case, avoiding importing this will be almost the same as reimplementing it.

It is MIT licensed. Is there any downside from adding this import? Per my commit message, it is already used in Tetragon - https://github.com/cilium/tetragon/blob/706409a5c4a351ac3aa10f5e17b3cd2f329a069b/pkg/filters/fields.go#L13